### PR TITLE
feat(bl-id-generator): endpoint for id generation

### PR DESCRIPTION
**Pull request by AdrianAndersen**

### CODE REVIEW

File: src/collections/unique-item/helpers/bl-id-generator.ts [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+      doc.image(pngBuffer, 0, canvas.height, { width: canvas.width });
+
+      resolve();
+    });
+    stream.on("error", reject);
+  });
+}
+
+async function generateBlIdPDF(): Promise<Buffer> {
+  const ids = generateBLIDs(400);
```
Comment (LarsSelbekk) [inline src/collections/unique-item/helpers/bl-id-generator.ts:132]: Make up your mind, either `BlId` or `BLID`

File: src/collections/unique-item/helpers/bl-id-generator.ts [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+  const buffers: Buffer[] = [];
+  const pass = new PassThrough();
+
+  doc.pipe(pass);
+
+  const idPromises: Promise<void>[] = ids.map((id: string) =>
+    addIdPageToDoc(id, doc)
+  );
+
+  await Promise.all(idPromises);
```
Comment (LarsSelbekk) [inline src/collections/unique-item/helpers/bl-id-generator.ts:144]: Have you verified (in the documentation) that it's safe to try to add many pages at the same time?

File: src/collections/unique-item/helpers/bl-id-generator.ts (hunk 1 above)
Comment (AdrianAndersen) [reply src/collections/unique-item/helpers/bl-id-generator.ts]: BlId it is

File: src/collections/unique-item/helpers/bl-id-generator.ts (hunk 2 above)
Comment (AdrianAndersen) [reply src/collections/unique-item/helpers/bl-id-generator.ts]: Hm, it seems like it just adds it in sequence anyways, so I changed it to forof

**Review by LarsSelbekk — APPROVED** [review]

**Pull request merged**